about.html

```diff
@@ -0,0 +1,116 @@
+                     The Paris Salon, open to the public from 1737, rapidly became the key factor in determining the reputation, 
+                     and so the price, of the French artists of the day. The Royal Academy in London, beginning in 1769, soon established a similar grip on the market,
+                      and in both countries artists put great efforts into making pictures that would be a success, 
+                      often changing the direction of their style to meet popular or critical taste.</p>
+            </div>
+            <p class="container sm-description text-center mb-5 params-exp">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis voluptates amet eum! Voluptate,
+                totam facere quae aut
+            </p>
+            <div class="line-empty"></div>
+        </div>
+    </div>
+    <div class="container section-product">
+        <h3 class="d-flex justify-content-center card-det">Artworks events - 2022
+           </h3>
+        <div class="container d-flex my-3 justify-content-center line-height"></div>
+        <p class="container text-center d-flex justify-content-center text-card">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque sapiente placeat nam corrupti dolores
+            culpa excepturi ducimus quasi magni illum. Quibusdam facere consequatur odio sit sint dolores numquam
+            iusto at?
+        </p>
+        <div class="logo-about d-flex justify-content-center"><img src="./img/artevent.png" alt="logo"></div>
+    </div>
+    <div class="pt-5 rank-section-about">
+        <h3 class="d-flex justify-content-center title-rank">We started in 2013</h3>
+        <div class="container d-flex my-3 justify-content-center line-height"></div>
+        <p class="container d-flex justify-content-center text-center rank-desc">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque sapiente placeat nam corrupti dolores
+            culpa excepturi ducimus quasi magni illum. Quibusdam facere consequatur odio sit sint dolores numquam
+            iusto at?
+        </p>
+
+        <div class="production-container container-fluid testymonies mb-5 testymonies2">
+        </div>
+    </div>
+    <footer class="container-fluid">
+        <div class="row">
+          <div class="col-md-3 py-5 text-center text-light footer-logo">© ART EXHIBITION</div>
+          <div class="col-md-9 p-5">
+            <h5 class="pb-3 mt-3 text-light footer-text-comment">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Pariatur voluptate voluptas id </h5>
+            <p class="text-light">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam, repellendus excepturi consequatur fuga molestias
+            atque harum eius debitis, dicta praesentium adipisci qui blanditiis tempora sequi possimus omnis, doloremque
+            assumenda ex.
+            </p>
+          </div>
+        </div>
+      </footer>
```

Comment:
- [ ] According to the original design, the about page should be shown the partners section and the footer section (with a white background)  on the `mobile version`, so kindly add the partners section and change the footer background to white on mobile version 💯👌


| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/181187487-85ce087d-5df6-4740-9ab9-096bf9d9229a.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/181187188-a206356f-c7c1-47a8-a2f7-c4e56b39c84e.PNG" width="500" /> |